docs/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+You can refer to [Kubernete's community documentation] on writing good commit messages, which provides good tips and ideas.
+
+Some PRs addess specific issues. Please, refer to the [CONTRIBUTING] documentation for more
+information on how to link a PR to an existing issue.
+
+[Kubernete's community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
+[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md
+
+# What environments does this code impact?
+
+- [ ] Cloud
+- [ ] Onprem
+- [x] None
+
+
+# How was this code tested?
+
+Please, select one or more if needed:
+
+- [ ] Local unit-tests
+- [ ] Local Kupe API subsystem tests
+- [ ] assisted-test-infra environment
+- [ ] dev-scripts environment
+- [ ] Reviewer's test appreciated
```

Comment:
> I would never expect a reviewer to run tests. The CI should always cover that

I agree that CI should always cover that but, realistically, this is not always the case. Flagging this doesn't mean the reviewer *has* to run the tests. But there are cases where this may be valuable.

I have tested other ppl's PRs in my environments and that has proven to be useful when CI is not covering the work being done.

Reply:
_How was this code tested?_ Is "manually" an option? Sometimes it's not possible/feasible to write automated tests. In such cases maybe we should let the developer describe a procedure for testing the particular.

Reply:
> How was this code tested? Is "manually" an option? Sometimes it's not possible/feasible to write automated tests. In such cases maybe we should let the developer describe a procedure for testing the particular.

I like this. To me, dev-script / test-infra is already a manual test but I see where you are coming from. Adding manually may encourage people to elaborate on how they did it.